### ISSUE TITLE
Does not work with svg images

### ISSUE THREAD

**Issue by stigzelm** [alpshq/craft-share-previews#11]
Currently not working when image is SVG format.
<img width="1269" alt="Screenshot 2022-03-01 at 23 17 00" src="https://user-images.githubusercontent.com/703669/156250692-155a6c92-09af-465a-be32-5fa47885a1f9.png">

**Comment by jakubjo** [comment]
That's true. I'll look into it and check if it's viable to add SVG support.
I'll keep you informed here.

**Comment by jripmeester** [comment]
@jakubjo Any update on this. Our client wants to incorporate logo's into the previews but they're used as SVG's in Craft. So that's a limitation at the moment.

**Comment by jakubjo** [comment]
Tagged `v2.1.0` with support for SVGs in asset layers. Make sure to install `inkscape` on your servers. It's required to prerender the SVG.

If you encounter any issue with specific SVGs, reply here or create a new issue.
